growthbook/growthbook
# [Docs] Add AI chatbot to the docs site

### Features and Changes
We recently started using Kapa.ai's AI chat capabilities to help with our customer support tickets. Kapa.ai also offers a chat widget that lives within a documentation site. This PR adds that chat widget to the GrowthBook docs site.

### Dependencies
None

### Testing
1. Pull my branch to your machine. 
2. Navigate to the `docs` directory (`$ cd docs`)
3. Run `$ yarn dev`
4. Navigate to http://localhost:3200/
5. Confirm the chat widget appears in the lower right corner (shown below)
6. Click on the chat widget and use the search bar to type in a question. The chat bot should return an answer based on our docs site.

### Screenshots

**EDITED on 12/20/2023** to replace screenshot. The old one had a GrowthBook icon; the new one below has a chatbot icon.

<img width="1435" alt="Screen Shot 2023-12-20 at 10 52 04 AM" src="https://github.com/growthbook/growthbook/assets/144278508/01d1733a-f55a-4249-91d2-21ab5c0e7110">
